_commands/os_x/brew.md

```diff
@@ -0,0 +1,109 @@
+---
+---
+
+brew
+--
+
+`brew`, or known by its longer name as Homebrew, bills itself as the "missing package manager for OS X". Here's an example of installing a package: 
+
+~~~
+$ brew install wget
+==> Downloading https://homebrew.bintray.com/bottles/wget-1.17.1.el_capitan.bottle.1.tar.gz
+Already downloaded: /Library/Caches/Homebrew/wget-1.17.1.el_capitan.bottle.1.tar.gz
+==> Pouring wget-1.17.1.el_capitan.bottle.1.tar.gz
+🍺  /usr/local/Cellar/wget/1.17.1: 9 files, 1.5M
```

Comment:
include an example of a package that you don't already have downloaded (uninstall and reinstall wget)